llvm/test/Transforms/LoopVectorize/PowerPC/optimal-epilog-vectorization.ll

```diff
@@ -1,4 +1,5 @@
 ; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 3
+; REQUIRES: PowerPC-registered-target
```

Comment:
```suggestion
; REQUIRES: powerpc-registered-target
```

I think these are case sensitive.

Reply:
It seems like the convention is to use lower case, but the case didn't impact the test outcome.

Reply:
The test won't fail, but it will be marked `Unsupported` even if you have the PPC backend enabled.

Reply:
Got it. I missed that earlier.